Harvest.Core/Services/SlothService.cs

```diff
@@ -81,11 +82,24 @@ public async Task<Result<SlothResponseModel>> MoveMoney(int invoiceId)
             {
                 return Result.Error("No accounts found for invoice: {invoiceId}", invoiceId);
             }
+
             if (_aeSettings.UseCoA)
             {
                 //Do an early check to make sure project account(s) are valid
                 foreach (var account in invoice.Project.Accounts)
                 {
+                    if(FinancialChartValidation.GetFinancialChartStringType(account.Number) == FinancialChartStringType.Invalid)
+                    {
+                        //Ok, it is an invalid COA format, so 99.99% chance this is still a KFS account. Lets try and convert it:

```

Comment:
probably an ok assumption, but do you think it's worth checking that the format is `3-` or i guess `{alpha}-` first before assuming KFS?